src/pages/Product/Product.js

```diff
@@ -29,6 +29,17 @@ const Product = () => {
   };
 
   useEffect(() => {
+
+    (async()=>{
+      try{
+        const res = await axios.post("/api/auth/login", {email:"adarshbalika@gmail.com", password:"adarshbalika"})
+        console.log("login",res);
```

Comment:
you can remove console.logs

Reply:
yeah that was for test purpose will do that